.github/workflows/sync_branch.yaml

```diff
@@ -0,0 +1,31 @@
+name: Sync storage-refactor
+on:
+  schedule:
+    - cron: 0 23 * * 1-5 # Week-daily at 23:00 UTC / 16:00 PST
+  workflow_dispatch: {} # Allow manual triggering
+
+jobs:
+  sync-storage-refactor:
+    runs-on: ubuntu-latest
+    name: Syncing storage-refactor
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Node
```

Comment:
Is this needed?

Reply:
I'm not sure, it is from the README of the Action: https://github.com/marketplace/actions/sync-branches. So I'm guessing yes?